lib/logstash/plugin_mixins/elasticsearch/api_configs.rb

```diff
@@ -52,6 +55,9 @@ module APIConfigs
         # The .cer or .pem file to validate the server's certificate
         :cacert => { :validate => :path },
 
+        # One or more hex-encoded SHA256 fingerprints to trust as Certificate Authorities
+        :ca_trusted_fingerprint => LogStash::PluginMixins::CATrustedFingerprintSupport,
```

Comment:
nit: if you have a chance to update the PR, it would be great to give plural naming since it contains a collection.
```suggestion
        :ca_trusted_fingerprints => LogStash::PluginMixins::CATrustedFingerprintSupport,
```

Reply:
The API is singular, the implementation silently allows plural as an advanced feature.